homeassistant/components/otbr/util.py

```diff
@@ -48,6 +49,17 @@
 )
 
 
+def compose_default_network_name(pan_id: int) -> str:
+    """Generate a default network name."""
+    return f"home-assistant-{pan_id:04x}"
```

Comment:
In testing it turns out this is too long, and gets rejected with 400 Bad Request by the OTBR :see_no_evil: 

The limit is 16 characters. We have to come up with something shorter.

Reply:
Let's drop the dash and use only the first byte of the PAN-ID

```suggestion
    return f"homeassistant-{pan_id >> 8:02x}"
```